src/sentry/static/sentry/app/views/onboarding/project/platformiconTile.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const PlatformiconTile = React.createClass({
+  propTypes: {
+    platform: React.PropTypes.string,
+    onClick: React.PropTypes.func
+  },
+
+  render() {
+    return (
+      <li
+        className={`platform-tile list-unstyled ${this.props.platform} ${this.props.platform.split('-')[0]} ${this.props.className}`}
```

Comment:
Prettier generated this super long line? Can we break this up and use some local variables (e.g. `platform`) above?